tests/test_game_state.py

```
@@ -0,0 +1,367 @@
+    original_score = game_state.score
+
+    # Simulate line clear
+    game_state._handle_line_clear(1)  # Single line
+
+    assert game_state.score > original_score
+    assert game_state.lines_cleared == 1
+
+
+def test_level_progression(game_state):
+    """Test level progression based on lines cleared."""
+    original_level = game_state.level
```

Ruff (0.14.2)

Local variable `original_level` is assigned to but never used

Remove assignment to unused variable `original_level`

(F841)